astropy/nddata/mixins/ndarithmetic.py

```diff
@@ -8,10 +8,16 @@
 
 import numpy as np
 
-from ... units import dimensionless_unscaled, UnitsError
+from ...units import dimensionless_unscaled, Quantity
```

Comment:
Astropy has always used relative imports.  Guido doesn't like them for some reason but other people disagree with him on that.